include/soc/bcm2835/raspberrypi-firmware.h

```diff
@@ -93,6 +93,8 @@ enum rpi_firmware_property_tag {
 	RPI_FIRMWARE_SET_GPIO_CONFIG =                        0x00038043,
 	RPI_FIRMWARE_GET_PERIPH_REG =                         0x00030045,
 	RPI_FIRMWARE_SET_PERIPH_REG =                         0x00038045,
+	RPI_FIRMWARE_GET_POE_HAT_VAL =                        0x00030049,
+	RPI_FIRMWARE_SET_POE_HAT_VAL =                        0x00030050,
```

Comment:
Out of curiosity, is there any chance to read back the revolutions per minute from the fan?

Reply:
It's just a simple fan switched by PWM on an attiny814. No way to read back the RPM.